crates/torii/grpc/proto/world.proto

```diff
@@ -78,3 +84,7 @@ message RetrieveEventsRequest {
 message RetrieveEventsResponse {
     repeated types.Event events = 1;
 }
+
+message Empty {
```

Comment:
What's this for? Can it be removed?

Reply:
basically i dont think we can return nothing from an rpc so what we do is just return an empty message. google does it the same way too. google/protobuf/empty.proto